dpypx/autodraw.py

```diff
@@ -99,7 +95,11 @@ async def check_pixel(self, x: int, y: int) -> bool:
 
         Returns True if the pixel was not already drawn.
         """
-        colour = self.grid[y - self.y0][x - self.x0]
+        pixel = self.grid[y - self.y0][x - self.x0]
+        colour = pixel[0]
+        if not pixel[1]:
```

Comment:
```suggestion
        colour, opaque = self.grid[y - self.y0][x - self.x0]
        if not opaque:
```
Using iterable unpacking for shorter code and better readability.